features/support/publishing_api.rb

```diff
@@ -1,17 +1,14 @@
 require 'gds_api/publishing_api_v2'
 require_relative '../../test/support/policy_tagging_helpers'
 
-def stub_publishing_api_v1_requests
-  v1_url = Plek.current.find('publishing-api')
-  stub_request(:any, %r{\A#{ v1_url }})
-end
+PUBLISHING_API_V1_ENDPOINT = Plek.current.find('publishing-api')
```

Comment:
I've updated this in 6db073d. I've kept the assignment (albeit to a variable now rather than a constant) as I think it makes it more explicit that we're dealing with a legacy V1 endpoint when we come back to remove it in the future. :innocent: